perseo-main/src/main/java/com/telefonica/iot/perseo/RulesServlet.java

```diff
@@ -160,26 +175,32 @@ protected void doDelete(HttpServletRequest request, HttpServletResponse response
         Utils.putCorrelatorAndTrans(request);
         response.setCharacterEncoding("UTF-8");
         response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        logger.info("delete rule " + request.getPathInfo());
-        //request.getPathInfo() returns null or the extra path information
-        //that follows the servlet path but precedes the query string and will
-        //start with a "/" character. So, we remove it with .substring(1)
-        String ruleName = request.getPathInfo();
-        if (ruleName == null) {
-            response.setStatus(400);
-            out.println("Deleting a rule require valid ruleName parameter");
-            out.close();
-            return;
+        
+        try {
+        	
```

Comment:
Blank line to be removed.

Reply:
Fixed in 61b8a4c9c7fe92f547917fd56b451a77f39db5f5